src/validation/index.ts

```diff
@@ -1,12 +1,3 @@
-export interface ValidationMessage {
-  key: string
-}
+export { Validation } from './validation'
 
-export interface Validation {
-  errors: Array<ValidationMessage>
-  fields: {
-    [name: string]: Validation
-  }
-  valid: boolean
-  warnings: Array<ValidationMessage>
-}
+export { Validator } from './validator'
```

Comment:
from index we need to export FieldValidators, ValidationResult,  FieldValidator, FactoryObcjets ,ValidationSeverity 

also, when exporting interfaces or type you must use export type

Reply:
ok